README.md

```diff
@@ -1,7 +1,7 @@
-AnitaB.org Portal - VMS project
+Volunteer Management System (VMS)
```

Comment:
```suggestion
AnitaB.org Portal - VMS project
```
I would suggest to undo the change made here since this is not included as part of the issue.

Reply:
@gaurivn I had updated the issue regarding this as well.